src/Template/Sadrs/xml/e2b.ctp

```diff
@@ -297,13 +308,13 @@
                     if ($sadrListOfDrug['suspected_drug']) echo 1 ;
                     else echo 2;
                 ?></drugcharacterization>
-                <medicinalproduct><?php echo $sadrListOfDrug['brand_name']; ?></medicinalproduct>
+                <medicinalproduct><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '', $sadrListOfDrug['brand_name']); ?></medicinalproduct>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate